WorkAndFun/src/main/resources/application.properties

```
@@ -1,11 +1,9 @@
 pl.infoshare.workandfun.users-path=WorkAndFun/src/main/resources/Users.json
 
 # Uwazaj by nie zmienic "update" na "create", bo wtedy przy uruchomieniu baza zostanie wyczyszczona i zostanie zaladowana pustka!
-spring.jpa.properties.hibernate.hbm2ddl.auto=update
+spring.jpa.properties.hibernate.hbm2ddl.auto=create
 
 # plik bedzie znajdowal sie w folderze "my_db", i bedzie mial nazwe "DB_1"
 spring.datasource.url=jdbc:h2:file:./my_db/DB_1;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
 spring.h2.console.enabled=true
 spring.h2.console.path=/console
```

```suggestion
spring.h2.console.path=/console
spring.mvc.hiddenmethod.filter.enabled=true
```
wpis by dodać PUT metodę do templatki